hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -18,9 +18,11 @@ filesets:
       - lowrisc:prim:lfsr
       - lowrisc:earlgrey_ip:pinmux_pkg
       - lowrisc:prim:assert
-      - lowrisc:prim:prim_pkg
       - lowrisc:prim:mubi
       - lowrisc:prim:multibit_sync
+      # TODO: prim_pkg is deprecated.
```

Comment:
We should probably create an issue to track this and mention the issue number here, right? (There are more prim_pkg-related TODO comments in other files)